models/PresentationContent.ts

```diff
@@ -42,6 +43,28 @@ class PresentationContent extends Base.BaseModel implements FFGenericContent {
 			}
 		});
 	}
+
+	// applies to questions only
+	public addReply(reply: FFTextContent, user:FFUser, cb:(PresentationContent)=>void) {
+
+		if (user) { reply.submitter = user; }
```

Comment:
The frontend will pass in "Anonymous User" as the user. If they aren't signed in, the null check will leave it as such instead of changing it to blank.